test/functional/base_test.go

```diff
@@ -230,6 +231,14 @@ func GetCinderVolume(name types.NamespacedName) *cinderv1.CinderVolume {
 	return instance
 }
 
+func GetCronJob(name types.NamespacedName) *batchv1.CronJob {
```

Comment:
in a follow up should add this to https://github.com/openstack-k8s-operators/lib-common/tree/main/modules/common/test/functional

Reply:
yes, also for keystone where this came from